README.md

```diff
@@ -1,410 +1,61 @@
-
-## Requirements
-
-To be able to use `instance_type` and `instance_gpu`, you'll need access to
-launch [instances from supported cloud vendors](#Supported-vendors). Please
-ensure that you have sufficient quotas with your cloud provider for the
-instances you intend to provision with Iterative Provider. If you're just
-starting out with a new account with a vendor, we recommend trying Iterative
-Provider with approved instances, such as the `t2.micro` instance for AWS.
-
-<details>
-<summary>Example with native AWS instace type and region</summary>
-<p>
-
-```tf
-terraform {
-  required_providers {
-    iterative = {
-      source = "iterative/iterative"
-    }
-  }
-}
-
-provider "iterative" {}
-
-resource "iterative_machine" "machine" {
-  region = "us-west-1"
-  image = "iterative-cml"
-  name = "machine"
-  instance_hdd_size = "10"
-  instance_type = "t2.micro"
-}
-```
-
-</p>
-</details>
-
-## Supported vendors
-
-The Iterative Provider currently supports AWS and Azure. Google Cloud Platform
-is not currently supported.
-
-<details>
-<summary>AWS instance equivalences</summary>
-<p>
-
-The instance type in AWS is calculated by joining the `instance_type` and
-`instance_gpu` values.
-
-| type | gpu  | aws         |
-| ---- | ---- | ----------- |
-| m    |      | m5.2xlarge  |
-| l    |      | m5.8xlarge  |
-| xl   |      | m5.16xlarge |
-| m    | k80  | p2.xlarge   |
-| l    | k80  | p2.8xlarge  |
-| xl   | k80  | p2.16xlarge |
-| m    | v100 | p3.xlarge   |
-| l    | v100 | p3.8xlarge  |
-| xl   | v100 | p3.16xlarge |
-
-| region   | aws        |
-| -------- | ---------- |
-| us-west  | us-west-1  |
-| us-east  | us-east-1  |
-| eu-north | us-north-1 |
-| eu-west  | us-west-1  |
-
-</p>
-</details>
-
-<details>
-<summary>Azure instance equivalences</summary>
-<p>
-
-The instance type in Azure is calculated by joining the `instance_type` and
-`instance_gpu`
-
-| type | gpu  | azure             |
-| ---- | ---- | ----------------- |
-| m    |      | Standard_F8s_v2   |
-| l    |      | Standard_F32s_v2  |
-| xl   |      | Standard_F64s_v2  |
-| m    | k80  | Standard_NC6      |
-| l    | k80  | Standard_NC12     |
-| xl   | k80  | Standard_NC24     |
-| m    | v100 | Standard_NC6s_v3  |
-| l    | v100 | Standard_NC12s_v3 |
-| xl   | v100 | Standard_NC24s_v3 |
+## Documentation
 
-| region   | azure       |
-| -------- | ----------- |
-| us-west  | westus2     |
-| us-east  | eastus      |
-| eu-north | northeurope |
-| eu-west  | westeurope  |
+You can find detailed documentation on how to configure and use the Iterative Provider [here](https://registry.terraform.io/providers/iterative/iterative/latest).
 
-</p>
-</details>
+## Support
 
-## The iterative-cml image
+If you find any issues or missing a feature? Let us know via [GitHub issues](https://github.com/iterative/terraform-provider-iterative/issues). Have questions? Join our [community on Discord](https://discord.com/invite/dvwXA2N), we'll be happy to help you get started!
```

Comment:
```suggestion
Have you found any issues or missing a feature? Let us know via [GitHub issues](https://github.com/iterative/terraform-provider-iterative/issues). Have questions? Join our [community on Discord](https://discord.com/invite/dvwXA2N), we'll be happy to help you get started!
```
Otherwise, the question mark would feel unnecessary (?)

Reply:
My bad, typo